cli/cmd/generate_gcp.go

```diff
@@ -0,0 +1,787 @@
+package cmd
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"time"
+
+	"github.com/imdario/mergo"
+	"github.com/spf13/cobra"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/pkg/errors"
+
+	"github.com/lacework/go-sdk/internal/file"
+	"github.com/lacework/go-sdk/lwgenerate/gcp"
+)
+
+var (
+	// Define question text here to be reused in testing
+	QuestionGcpEnableConfig            = "Enable Config Integration?"
+	QuestionGcpEnableAuditLog          = "Enable AuditLog Integration?"
+	QuestionGcpOrganizationIntegration = "Organization Integration?"
+	QuestionGcpOrganizationID          = "Specify the GCP Organization ID:"
+	QuestionGcpProjectID               = "Specify the Project ID to be used to provision Lacework resources:"
+	QuestionGcpServiceAccountCredsPath = "Specify Service Account credentials JSON path: (optional)"
```

Comment:
```suggestion
	QuestionGcpServiceAccountCredsPath = "Specify service account credentials JSON path: (optional)"
```

Reply:
I think you get the pattern. We need to talk with product and docs team to check that these end-user messages are correct.

I am going to update the AWS generate command to AVOID using capitalized words where they are not needed.